components/org.wso2.identity.webhook.wso2.event.handler/src/main/java/org/wso2/identity/webhook/wso2/event/handler/api/builder/WSO2UserOperationEventPayloadBuilder.java

```diff
@@ -179,25 +220,27 @@ private List<User> buildUserList(AbstractUserStoreManager userStoreManager, Map<
         return users;
     }
 
-    private static void enrichUser(AbstractUserStoreManager userStoreManager, String userName, User user)
+    private static void enrichUser(AbstractUserStoreManager userStoreManager, String domainQualifiedUserName, User user)
             throws IdentityEventException {
 
         String userId;
         try {
-            userId = userStoreManager.getUserClaimValue(userName, FrameworkConstants.USER_ID_CLAIM,
+            userId = userStoreManager.getUserClaimValue(domainQualifiedUserName, FrameworkConstants.USER_ID_CLAIM,
```

Comment:
Are we getting the user id from the user store here ?

Reply:
Yes